docs/pages/interfaces.rst

```diff
@@ -38,52 +38,58 @@ for example, can have one, two or three possible types. See the example below:
 Mappable
 --------
 
-Something is considered mappable if we can `map` it using a function, generally
-`map` is a method that accepts a function. An example in this library is
-:class:`Maybe <returns.maybe.Maybe>`, it has the `map` method.
+Something is considered mappable if we can ``map`` it using a function,
+generally ``map`` is a method that accepts a function.
 
-:class:`Mappable <returns.interfaces.mappable.MappableN>`
-interface help us to create our own mappable container like
-:class:`Maybe <returns.maybe.Maybe>`.
+An example in this library is :class:`Maybe <returns.maybe.Maybe>`,
+that implements the ``Mappable`` interface:
 
 .. code:: python
 
-  >>> from typing import Any, Callable, TypeVar
+  >>> from returns.maybe import Maybe, Some
+
+  >>> def mappable(string: str) -> str:
+  ...     return string + '!'
+
+  >>> maybe_str: Maybe[str] = Maybe.from_value('example')
```

Comment:
It is better to use the same constructors on the both sides: `Maybe.from_value` or `Some`.

So:

```python
maybe_str: Maybe[str] = Some('example')
assert maybe_str.map(mappable) == Some('example!')
```

Reply:
Done!